web/src/hooks/useShare.ts

```diff
@@ -21,12 +21,14 @@ export function useShare() {
   const share = useCallback(
     async (shareData: ShareOptions, callback?: (argument: string) => void) => {
       try {
-        await CapShare.share(shareData);
+        const result = await CapShare.share(shareData);
+        return result;
       } catch (error) {
         if (error instanceof Error && !/AbortError|canceled/.test(error.toString())) {
           console.error(error);
           callback?.(t('share-button.share-error'));
         }
+        return null;
```

Comment:
Minor nit, I don't think you need to return null here, just return will do (null vs undefined) but it's not a problem either.